sismic/io/yaml.py

```diff
@@ -13,16 +13,18 @@
 
 class SCHEMA:
     contract = {schema.Or('before', 'after', 'always'): schema.Use(str)}
-
-    transition = {
+    # initial transitions can't have a guard and do not need a priority
+    initial_transition = {
         schema.Optional('target'): schema.Use(str),
+        schema.Optional('action'): schema.Use(str),
```

Comment:
Initial transitions can have contracts as well (I know this is not part of scxml nor uml, but I don't see any practical reason to not have them ;))

Reply:
Okay I meant to discuss that with you. I’ll add that to the scheme.